pages/wiki/porting-status.md

```diff
@@ -21,6 +21,7 @@ The WearOS smartwatches are the most widespread and easy to support. The source
 - LG G Watch R *(maintained by atx)*
 - LG Watch Urbane *(maintained by TheAppleMan & kido)*
 - LG Watch W7 *(maintained by dodoradio & MagneFire)*
+- Moto 360 (1nd generation) *(maintained by MagneFire)*
```

Comment:
typo: `1nd` != first

Reply:
Great catch! No idea how I missed this :laughing: